estate/models/estate_model.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+
+from odoo import fields, models
+
+
+class EstateModel(models.Model):
```

Comment:
```suggestion
class EstateProperty(models.Model):
```
Following previous comment.